client/packages/common/src/styles/theme.ts

```diff
@@ -138,6 +139,7 @@ export const themeOptions = {
     gradient: {
       primary: 'linear-gradient(156deg, #f80 4%, #e63535 96%)',
       secondary: 'linear-gradient(156deg, #78a3fc 4%, #3e7bfa 96%)',
+      tertiary: 'linear-gradient(156deg, #e63535 4%, #78a3fc 96%)',
```

Comment:
Hmm, since initialisation and login has different gradient, maybe server discovery should too ?

Reply:
https://github.com/openmsupply/open-msupply/issues/58